### PR TITLE
Update package management with Glide

### PR DESCRIPTION
**Pull request by EmileVauge**
No more Godep!!!
Please welcome Glide :)
This resolves https://github.com/EmileVauge/traefik/issues/35

### CODE REVIEW

**Comment by ldez** [comment]
mj-side-glide-o

File: build.Dockerfile [hunk 1]
```diff
@@ -15,17 +18,11 @@ RUN set -ex; \
...
-
-RUN mkdir Godeps
-COPY Godeps/Godeps.json Godeps/
-RUN godep restore
+COPY glide.yaml glide.yaml
+RUN glide up --update-vendored
 
 COPY . /go/src/github.com/emilevauge/traefik
+
+ENV GOPATH /go
```
Comment (vdemeester) [inline build.Dockerfile]: Not sure this is needed (should be set by the parent image `golang`)

Comment (EmileVauge) [inline build.Dockerfile]: You're right, corrected

**Comment by vdemeester** [comment]
friends-ross-do-the-dance

**Comment by vdemeester** [comment]
By the way, looks good, seems good, works good so LGTM 🐹.

**Pull request merged**